core/src/main/resources/application.yml

```diff
@@ -446,6 +446,7 @@ cb:
   custom.user.data: |
     touch /tmp/custom-user-data
     date >> /tmp/custom-user-data
+  upgrade.enablecustomtemplate: true
```

Comment:
Can you please remove this option anyways? Let's allow all custom blueprints. I'm not sure what the end behavior will be with custom blueprints, but I don't think a deployment wide option will help.